.github/workflows/e2e.yml

```diff
@@ -57,9 +57,16 @@ jobs:
           export E2E_DOCKER_CONFIG_JSON_SECRET=`cat ~/.docker/config.json| base64 -w 0`
           make e2e E2EARGS="-v=5 -skip-features 'pv expansion|custom config for dynamic|tools for exporter'"
         env:
+          E2E_AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          E2E_AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          E2E_BR_IMAGE: reg.vesoft-inc.com/cloud-dev/br-ent
```

Comment:
Why put it in `cloud-dev`?

Reply:
Currently br-ent is in cloud-dev. @MegaByte875 to move them into the CI folder